UI/data/locale/en-US.ini

```diff
@@ -559,6 +559,8 @@ VolControl.SliderUnmuted="Volume slider for '%1':"
 VolControl.SliderMuted="Volume slider for '%1': (currently muted)"
 VolControl.Mute="Mute '%1'"
 VolControl.Properties="Properties for '%1'"
+VolControl.UnassignedWarning.Title="Unassigned Audio Source"
+VolControl.UnassignedWarning.Text="\"%1\" is not assigned to any audio tracks and it will not be audible in streams or recordings.\n\nTo assign an audio source to a track, open the advanced audio properties via the right-click menu or the cog button in the mixer dock toolbar."
```

Comment:
```suggestion
VolControl.UnassignedWarning.Text="\"%1\" is not assigned to any audio tracks and it will not be audible in streams or recordings.\n\nTo assign an audio source to a track, open the Advanced Audio Properties via the right-click menu or the cog button in the mixer dock toolbar."
```

Reply:
Updated.